src/repos.js

```diff
@@ -1,21 +1,22 @@
-export default {
-  'sivakumar-kailasam/github-webook-test-repo': {
-    labels: [
-      'help wanted',
-      'a11y',
-      'bug',
-      'question',
-    ]
-  },
-  'acorncom/ember-hitlist-tester': {
-    labels: [
-      'Help Wanted'
-    ]
-  },
-  'emberjs/ember.js': {
-    labels: [
-      'Needs Help',
-      'Good for New Contributors'
-    ]
-  }
-};
+// NOTE: Ensure the labels are in lower case to make it easier for repo owners
+const standardLabels = [
+  'bug',
+  'help wanted',
+];
+
+let repoList = [
+  'ember-learn/ember-help-wanted',
+  'ember-learn/ember-help-wanted-webhook',
+  'emberjs/guides',
+  'emberjs/website',
+  'emberjs/ember.js',
+  'ember-cli/ember-cli',
+];
+
+let repoHash = {};
+
+repoList.map(repo => {
+  repoHash[repo] = { labels: standardLabels };
```

Comment:
Hmm, I like the idea of using standard labels, but would also like to have an "out" for repos that don't use the labels we're using yet.  I don't want to force folks to have to adopt our label standards in order to use the tool. Thoughts?